README.md

```diff
@@ -384,6 +409,29 @@ EXAMPLE
 
 _See code: [src/commands/domain/unpublish.js](https://github.com/SmartBear/swaggerhub-cli/blob/v0.2.12/src/commands/domain/unpublish.js)_
 
+## `swaggerhub domain:visibility`
+
+Set visibility of a Domain
+
+```
+USAGE
+  $ swaggerhub domain:visibility OWNER/DOMAIN_NAME/[VERSION]
+
+ARGUMENTS
+  OWNER/DOMAIN_NAME/[VERSION]  Domain Identifier
+
+OPTIONS
+  -h, --help  show CLI help
+  --private   sets the default API version as private
```

Comment:
These flags shouldn't both exist, perhaps "--visibilty=private" ? 
Since if they both cannot be required, but at least one is (makes it complicated to state that). It's also not clear if they are both present, which has priority.
UPDATE
I see that you can set them as mutually exclusive, but at least one of them must be required?

Reply:
Both required but one of them is exclusive to another one. So it works. I just think using flag is easier than a flag with value